api/modelss/build-log.js

```diff
@@ -0,0 +1,24 @@
+const associate = ({ BuildLog, Build }) => {
+  BuildLog.belongsTo(Build, {
+    foreignKey: "build",
+    allowNull: false,
+  })
+}
+
+module.exports = (sequelize, DataTypes) => {
+  const BuildLog = sequelize.define("BuildLog", {
+    output: {
+      type: DataTypes.STRING,
+    },
+    source: {
+      type: DataTypes.STRING,
+    },
+  }, {
+    tableName: "buildlog",
+    classMethods: {
+      associate,
```

Comment:
Is there a reason to define the associations in classMethods like this instead of as described in the [Sequelize docs](http://docs.sequelizejs.com/en/latest/docs/associations/#creating-elements-of-a-belongsto-has-many-or-hasone-association)? I don't see anything in the docs indicating we have to do something different to define associations for [legacy tables](http://docs.sequelizejs.com/en/latest/docs/legacy/).

Is there an advantage to doing it this way? It seems to rely on the reader knowing/figuring out how classMethods work on top of things like `BuildLog.belongsTo(...)` instead of simply defining the relationship inline.

Everything else in this PR looks good. I cross-referenced the types with our sails models.